lib/consul/async/consul_template.rb

```diff
@@ -148,7 +170,7 @@ def render(tpl, tpl_file_path, params = {})
           current_erb_path: tpl_file_path,
           params: params
         }
-        result = ERB.new(tpl).result(binding)
+        result = ERB.new(tpl, nil, '<>-%').result(binding)
```

Comment:
I'm not sure it won't break the consul stuff, must check before

Reply:
Remove it unless necessary, you may add it later in another review if it is better, keep this patch compact

Reply:
Roger.